public/docs/ts/latest/guide/component-communication.jade

```diff
@@ -0,0 +1,707 @@
+  
+:marked
+  When we run the app and click the **Invite heroes** button, the application UI reflects this stucture:
+
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/heroes-invited-ui.png" alt="Heroes are invited")
+  
+:marked
+  ### Using `@Input`
+  
+  `Input` is defined in the core module of Angular, and we need to import it so that we can use it:
+  
++makeExample('component-communication/ts/app/invite-heroes/hero-panel.ts', 'import', null, {blk: /(Input)/g})
+
+:marked
+  `@Input()` always must be used with parentheses:
+
++makeExample('component-communication/ts/app/invite-heroes/hero-panel.ts', 'input-def', null, {pnk: /(@Input\(\))/g})
+
+:marked
+.alert.is-critical
+  :marked
+    It's easy to forget the parentheses. Our application will fail mysteriously if we do.
+:marked
+  Instead of using the `@Input()` annotation, we can use the `inputs` annotation property of `@Component()` to specify input properties.
+
+  Using the alternative method, we could have defined the `hero` input property this way:
+  
++makeExample('component-communication/ts/fragments/using-inputs.ts', null, null, {pnk: /(inputs)/g})
+
+:marked
+.alert.is-helpful
+  :marked
+    In this chapter, we are age going to use the `@Input()` annotation.
+:marked
+  ### The convenience of input data binding
+  
+  The change detection mechanism of Angular makes input data binding easy-to-use. We can utilize the input property values in expressions, too,
+  and it makes displaying data simple.
+  
+  We have already implemented the functionality behind the __Invite heroes__ function with input data binding. The same mechanism allows us
+  to implement announcing jobs. As the altered definition of `HeroPanel` shows, now it has another input property, `request`, which has two
+  occurrences within the template:
+
++makeExample('component-communication/ts/app/send-job-request/hero-panel.ts', 'component', 'send-job-request/hero-panel.ts (extract)', {pnk: /(&quot;request&quot;|{{request \|\|.*\}\})/g})
+
+:marked
+  As the value of `request` changes, the value of the `class` attribute is updated, as well as the text of the `<h4>` tag through the 
+  `request || 'No job announced'` expression. The `HeroJobBoard` side of the input data binding is as simple as we expect:
+  
++makeExample('component-communication/ts/fragments/sending-request.ts', null, 'send-job-request/hero-job-board.ts (extract)', {pnk: /(\[request\]=request)/g})
+
+:marked
+  After inviting heroes, clicking the **Ask** button invokes `announceJob()` that sets the `request` property of `HeroJobBoard` &mdash; and Angular propagates this
+  change to all `HeroPanel` instances:
+
+figure.image-display
+  img(src="/resources/images/devguide/component-communication/send-job-request-ui.png" alt="Job request is sent")
+  
+:marked
+  Now, let's see how a child can communicate with its parent.
```

Comment:
Set the problem up.

Notice that in this illustration we did not provide a way for the child to accept a job. We could have bound a callback method  with a property binding
>and may we should show that?
but that's hacky.

Child components don't use property binding to communicate with their parents. They can talk to their parents with events.